### PR TITLE
[12.0][ADD] hr_employee_health: track specific needs of employees

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
No test-cases since this plugin is purely about extra data fields and no functions to test

**Review by pedrobaeza — COMMENTED** [review]
As these are very protected personal data, you should put security at field level (https://www.odoo.com/documentation/11.0/reference/security.html#field-access) for avoiding leaks of information through XML-RPC or so.

**Comment by alexey-pelykh** [comment]
@pedrobaeza thanks! Fixed

**Comment by alexey-pelykh** [comment]
Eventually, hopefully, a self-service employee feature will be around

File: hr_employee_health/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        'Braibean Apps, '
+        'Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'installable': True,
+    'application': False,
+    'summary': 'Allows storing information about employee\'s health',
+    'depends': [
+        'hr',
+    ],
+    'data': [
```
Comment (nikul-serpentcs) [inline hr_employee_health/__manifest__.py:19]: @alexey-pelykh IMO manage sequence as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)

File: hr_employee_health/data/data_blood_type.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```
Comment (nikul-serpentcs) [inline hr_employee_health/data/data_blood_type.xml]: `<odoo noupdate="1">`

File: hr_employee_health/data/data_health_condition_severity.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```
Comment (nikul-serpentcs) [inline hr_employee_health/data/data_health_condition_severity.xml]: same here

File: hr_employee_health/data/data_health_condition_type.xml [hunk 4]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```
Comment (nikul-serpentcs) [inline hr_employee_health/data/data_health_condition_type.xml]: same here

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

File: hr_employee_health/__manifest__.py (hunk 1 above)
Comment (alexey-pelykh) [reply hr_employee_health/__manifest__.py:19]: fixed

File: hr_employee_health/data/data_blood_type.xml (hunk 2 above)
Comment (alexey-pelykh) [reply hr_employee_health/data/data_blood_type.xml]: fixed

File: hr_employee_health/data/data_health_condition_severity.xml (hunk 3 above)
Comment (alexey-pelykh) [reply hr_employee_health/data/data_health_condition_severity.xml]: fixed

File: hr_employee_health/data/data_health_condition_type.xml (hunk 4 above)
Comment (alexey-pelykh) [reply hr_employee_health/data/data_health_condition_type.xml]: fixed

**Comment by alexey-pelykh** [comment]
@nikul-serpentcs please review the changes requested

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

**Comment by alexey-pelykh** [comment]
@luistorresm I'd like to request your review as well

**Review by gurneyalex — APPROVED** [review]

**Comment by alexey-pelykh** [comment]
@pedrobaeza is this PR good to be merged (2 approves + 5 days have passed)?

**Pull request merged**